src/main/kotlin/com/intershop/gradle/icm/docker/extension/DevelopmentConfiguration.kt

```diff
@@ -256,6 +266,20 @@ open class DevelopmentConfiguration
         config
     }
 
+    val intershopEnvironmentProperties: EnvironmentProperties by lazy {
+        val envConfig = EnvironmentProperties(objectFactory)
+        val keys = configProperties.keys.filter {it.toString().startsWith(Configuration.INTERSHOP_ENVIRONMENT_PREFIX)}
+            .stream().map { it.toString()}
+        keys.forEach {
+            val p = getConfigProperty(it)
+            it.replaceFirst(Configuration.INTERSHOP_ENVIRONMENT_PREFIX, ENV_PREFIX)
+                .replace(".", "_")
```

Comment:
When providing properties using environment variables starting with `ISH_ENV_` it is not necessary to replace `.` by `_` and converting to upper case.